internal/armada/configuration/types.go

```diff
@@ -113,7 +113,11 @@ type SchedulingConfig struct {
 	DefaultJobTolerationsByResourceRequest map[string][]v1.Toleration
 	// Maximum number of times a job is retried before considered failed.
 	MaxRetries uint
-	// Weights used when computing fair share.
+	// Controls how fairness is calculated. Can be either AssetFairness or DominantResourceFairness.
```

Comment:
This list of possible values is bound to become out of date.

Reply:
Once DRF is used everywhere, I want to remove asset fairness and leave DRF as the only option.